src/components/user-form/UserForm.jsx

```diff
@@ -1,28 +1,25 @@
 import React, { useCallback, useState } from 'react';
 import PropTypes from 'prop-types';
 import Input from '../input/Input';
+import Form from '../form/Form';
 import Button, { ButtonType } from '../button/Button';
 import styles from './UserForm.module.css';
 
-export default function UserForm({
-  initialValues,
-  onSubmit,
-}) {
-  const [username, setUsername] = useState(initialValues ? initialValues.username : '');
+export default function UserForm({ initialValues, onSubmit }) {
+  const [username, setUsername] = useState(
+    initialValues ? initialValues.username : '',
```

Comment:
Mhh, I was thinking, if we use apply here optional chaining with null coalescing this can become:

```js
initialValues?.username ?? ''
```

Or am I wrong?